src/App.jsx

```diff
@@ -14,33 +16,127 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [state, setState] = useState({
+    sortedGoods: [...goodsFromServer],
+    sortedAlpha: [...goodsFromServer].sort((a, b) => a.localeCompare(b)),
+    sortedLength: [...goodsFromServer].sort((a, b) => a.length - b.length),
```

Comment:
Rafael, why are you storing 3 versions of goodsFromServer in your component's state?
A better practice would be to keep a single array in state and alter this array as needed for the rendering.
Your code is working, but there is a lot of re-work here, don't you think?
Try keeping only one array in your state and creating functions for altering that array. Then, you can call the functions on button click accordingly.